cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -192,16 +197,26 @@ function deploy_sriov_operator {
     curl -L https://github.com/openshift/sriov-network-operator/archive/${OPERATOR_GIT_HASH}/sriov-network-operator.tar.gz | tar xz -C ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/
   fi
 
+  if [ ! $RELEASE_VERSION = "4.4.0" ]; then
```

Comment:
Note
* This if can be removed as the code here is generic for all versions i saw.
* We can also use diff file, but apply it only in the first time the folder is created.
* Another method is to expend the make target, to the one it uses, skipping the ENABLE_ADMISSION_CONTROLLER=false in it.
(it wouldn't solve the need of the other lines for now).